tools/benchmarks/generate_ftl_file.py

```diff
@@ -0,0 +1,157 @@
+import argparse
```

Comment:
I'm happy to drop this commit entirely if the script isn't useful, but I thought it was worth offering up (and am happy to make fixups).